owid/catalog/meta.py

```diff
@@ -72,6 +72,18 @@ def to_dict(self) -> Dict[str, Any]:
 @dataclass_json
 @dataclass
 class VariableMeta:
+    """Allowed fields for `display` attribute used for grapher:
+        name
+        zeroDay
+        yearIsDay
+        includeInTable
+        numDecimalPlaces
+        conversionFactor
+        entityAnnotationsMap
+    Fields `unit` and `shortUnit` are copied from attributes `unit` and `short_unit`
+    on VariableMeta object
+    """
+
```

Comment:
Wouldn't it be better to have snake-case names for the display dictionary (that then get translated into camel-case, grapher names)? I just suggest it for consistency, but feel free to disregard it (I suppose it would need to be changed in some other places too).

Reply:
Good question... I don't know, as you say, it's a tradeoff. I prefer `camelCase` as it reminds us we're working on something that is not native to `etl`, but is part of a different system (`grapher`). It's a weak argument though. Anyway, less code is better so I'm keeping `camelCase` and adding a note ;).